Source/SplineFollow/Public/Components/SFSplineMarkers.h

```diff
@@ -75,6 +75,7 @@ struct SPLINEFOLLOW_API FSFSplineMarkerInfos
 
     FSFSplineMarkerInfos() :
         Type( ESFSplineMarkerType::Single ),
+        bNormalizedDistance( 1 ),
```

Comment:
I would init it with "true" instead of "1"

Reply:
done